packages/@okta/i18n/src/properties/login_zh_TW.properties

```diff
@@ -192,7 +192,7 @@ errors.E0000110 = \u60A8\u5B58\u53D6\u7684\u9023\u7D50\u5DF2\u904E\u671F\u6216\u
 errors.E0000111 = {0} \u662F\u552F\u8B80\u7269\u4EF6\uFF0C\u56E0\u6B64\u7121\u6CD5\u4FEE\u6539\u3002
 errors.E0000112 = \u6B64\u4F7F\u7528\u8005\u4ECD\u8655\u65BC\u555F\u7528\u72C0\u614B\uFF0C\u56E0\u6B64\u7121\u6CD5\u66F4\u65B0\u3002\u8ACB\u5728\u5E7E\u5206\u9418\u5F8C\u518D\u8A66\u4E00\u6B21\u3002
 # {0} is a server-side error response, oftentimes coming directly from a vendor.
-errors.E0000113 = {0}\u3002
+errors.E0000113 = {0}.
```

Comment:
Is this period `.` used in Chinese?

Reply:
I reverted all the changes that replaced unicode with a dot or comma, we should be good now